testprojects/src/python/interpreter_selection/resolver_blacklist_testing/BUILD

```diff
@@ -0,0 +1,18 @@
+python_binary(
+  name='test_bin',
+  source='main.py',
+  compatibility=['CPython>3'],
+  dependencies=[
+    ':reqlib'
+  ]
+)
+
+# Without blacklisting, pex resolver will error out
+# on a transitive requirement of jupyter (functools32)
+# when resolving for Python 3.
+python_requirement_library(
+	name='reqlib',
+	requirements=[
+		python_requirement('jupyter')
+	]
+)
```

Comment:
The spacing here is confusing me -- I would assume this would be indented only two spaces? I also don't know if it's the style of this repo or some other one, but I feel like I usually see BUILD files using commas at the end of the last element in a list, e.g.:

```python
python_requirement_library(
  name='reqlib',
  requirements=[
    python_requirement('jupyter'), # comma here
  ], # and here
)
```

I do this typically only when elements of the list are aligned vertically (as they are here). It's not necessary.